aioelasticsearch/helpers.py

```diff
@@ -147,3 +150,121 @@ def _update_state(self, resp):
         self._successful_shards = resp['_shards']['successful']
         self._total_shards = resp['_shards']['total']
         self._done = not self._hits or self._scroll_id is None
+
+
+async def worker_bulk(client, datas: List[dict], actions: List[str],  **kwargs):
```

Comment:
All project do not provide yet typings everywhere, so it not makes to put them here. Later typings will be provided everywhere...